src/ui/server/controllers/root/cookies/index.ts

```diff
@@ -54,7 +54,7 @@ export const post = (req: Request, res: Response) => {
 
   return res.render(TEMPLATE, {
     userName: getUserNameFromSession(req.session.user),
-    ...singleInputPageVariables({ ...PAGE_VARIABLES, BACK_LINK: backLink, START_ROUTE: startRoute }),
+    ...singleInputPageVariables({ ...PAGE_VARIABLES, BACK_LINK: backLink, START_ROUTE: startRoute, FEEDBACK_ROUTE: LINKS.EXTERNAL.FEEDBACK }),
```

Comment:
Looks like we don't need the `FEEDBACK_ROUTE` here because `singleInputPageVariables` has it?